examples/pxScene2d/src/pxAnimate.h

```diff
@@ -85,28 +108,25 @@ class pxAnimate: public rtObject
         rtError to(double& v)   const { v = (uint32_t) mTo; return RT_OK;   }
         rtError duration(double& v)   const { v = mDuration; return RT_OK; }
         rtError count(int32_t& v)   const { v = mCount; return RT_OK;   }
-        rtError status(rtString& v)   const { v = mStatus; return RT_OK;   }
+        rtError status(rtString& v)   const { v = mapStatus(mStatus); return RT_OK;   }
         rtError cancelled(bool& v)   const { v = mCancelled; return RT_OK;   }
    
-        rtString mStatus;
+        uint32_t mStatus;
         int32_t mCount;
         bool mCancelled;
         double mDuration;
         double mFrom;
         double mTo;
     };
-   
   private:
-    rtString mapStatus(uint32_t v);
-
     rtObjectRef mProps;
     rtObjectRef mCurrDetails;
     uint32_t mInterp;
     pxConstantsAnimation::animationOptions mType;
     double mProvisionedDuration;
     int32_t mProvisionedCount;
     bool mCancelled;
-    rtString mStatus;
+    uint32_t mStatus;
```

Comment:
`mStatus` is clearly an `enum` ...  which is a good thing ... strongly typed.  Please use that `enum` or a `typedef` of it instead of `uint32_t` wherever you're using or passing it about